openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/WheelchairAttributesGraphStorage.java

```diff
@@ -141,13 +151,15 @@ public void setEdgeValues(int edgeId, WheelchairAttributes attrs) {
 
 		encodeAttributes(attrs, buffer);
 
+
 		orsEdges.setBytes(edgePointer + efWheelchairAttributes, buffer, BYTE_COUNT);
+
 	}
 
 	private void encodeAttributes(WheelchairAttributes attrs, byte[] buffer) {
```

Comment:
Use Graphhopper's EncodedValues properly instead of all this bit-shifting patterns

Reply:
see above